newIDE/app/src/Profile/ProfileDialog.js

```diff
@@ -87,53 +87,56 @@ const ProfileDialog = ({
           <Tab label={<Trans>Games Dashboard</Trans>} value="games-dashboard" />
         </Tabs>
       }
+      flexBody
```

Comment:
I guess you could have used flexColumnBody that's not really important

Reply:
good catch